### PR TITLE
[JENKINS-54599] - Prepare the master branch towards the Jetty upgrade in the plugin

### PR DESCRIPTION
**Pull request by oleg-nenashev**
https://issues.jenkins-ci.org/browse/JENKINS-54599

@jenkinsci/java11-support

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
No Tests found...

File: Jenkinsfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ for (int i = 0; i < platforms.size(); ++i) {
 
                 stage('Archive') {
                     /* Archive the test results */
-                    junit '**/target/surefire-reports/TEST-*.xml'
+                    // junit '**/target/surefire-reports/TEST-*.xml'
```
Comment (alecharp) [inline Jenkinsfile]: could we just use `allowEmptyResults` so that this is not commented and still not failing if there is no test?

**Review by alecharp — CHANGES_REQUESTED** [review]

**Review by alecharp — APPROVED** [review]
LGTM

**Comment by alecharp** [comment]
yes ok, I looked at the attribute, but missed the `false`. :man_facepalming:

**Comment by oleg-nenashev** [comment]
As you may see, me too :)

**Pull request merged**